### DISCUSSION TITLE
Collecting feedback on the ADF webpage

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#82] [TF4: Diagnostics, output]
The [ADF](https://github.com/NCAR/ADF) is the new diagnostic package that is going replace the AMWG diag. 
ADF can now produce comparison against another run and against obs.  

For run https://github.com/NCAR/amwg_dev/issues/78, we have:

- [ADF diags](https://webext.cgd.ucar.edu/FWscHIST/f.e21.FWscHIST.ne30_L48_BL10_cam6_3_041_kzz3_zmtop75.hf.001/atm/ADF/) 
- [AMWG diags](https://webext.cgd.ucar.edu/FWscHIST/f.e21.FWscHIST.ne30_L48_BL10_cam6_3_041_kzz3_zmtop75.hf.001/atm/f.e21.FWscHIST.ne30_L48_BL10_cam6_3_041_kzz3_zmtop75.hf.001.1980_1989-f.e21.FWscHIST.ne30_L48_BL10_cam6_3_041_control.hf.001.1980_1989/)

The ADF team is requesting feedback on the webpage produced by ADF. Please post your feedback at: https://github.com/NCAR/ADF/discussions/114

**Comment by swrneale** [comment]
@cecilehannay My comments would be:
-Have the plot show on the right side while the list of links remains on the left
-Maybe have a 'top ten' grouping so that we can quick look at the most frequent plots (PRECT, SWCF, LWCF,TS, etc.)
-Colorizing the tables red/green to show which run of the two is better compared to obs., the land do something like this and we had some simple tables doing this in the orig. diags.
-In the 'links' it would be good to be able to go to other milestone diagnostics (CAM6/CESM2/20th/FHIST etc.)
Looking good though!
Rich

**Reply by justin-richling** [reply]
Can you please point me to an example of a colorized table you were mentioning earlier in the CAM7 meeting?

**Reply by swrneale** [reply]
The old diagnostics package creates an ncl table and displays a png of that on the webpage. It is also possible to colorize the background of each cell in html on the table webpage, but I think the best option is to create the actual table in python and then display that as a png, this gives us more flexibility to do other things I think. There are a few packages to export a stylized pandas dataframe to an image e.g.,(https://pypi.org/project/dataframe-image/)

**Comment by swrneale** [comment]
Simple example of multi-panel view of figures using SCAM.
http://wwwint.cgd.ucar.edu/project/amp/rneale/scam/web/cesm2_zm01-cesm2_zm00-ARM95/index.html

**Comment by adamrher** [comment]
I'm wondering if we should introduce a pattern correlation to the top of the anomaly map plots, in addition to the mean and rmse? I've found it a useful metric for evaluating the skill of my CLUBB+MF runs. 

I use the ncl routine pattern_cor: https://www.ncl.ucar.edu/Document/Functions/Contributed/pattern_cor.shtml. I'm sure there's a python library with it out there somewhere.

**Comment by cecilehannay** [comment]
**Please post your feedback at: https://github.com/NCAR/ADF/discussions/114 as requested in the initial post** 

I am sorry to be a pain. But it is best if you post the feedback on the corresponding ADF webpage: https://github.com/NCAR/ADF/discussions/114. Some ADF folks don’t follow the amwg_dev repo. Thanks.

**Comment by cecilehannay** [comment]
I am sorry to be a pain. But it is best if you post these on the corresponding ADF webpage.Some ADF folks don’t see follow this repo. 

I reposted Rich’s earlier comments on the ADF.  And I can repost these too.  But if is easier if it goes right away at the right spot.  Thanks.
